changelogs/fragments/850-proxmox_kvm-remove_hard_coded_defaults.yml

```
@@ -0,0 +1,3 @@
+breaking_changes:
+- proxmox - remove hard_coded defaults which are different from proxmox or will be changed later by proxmox (https://github.com/ansible-collections/community.general/pull/850).
+- proxmox_kvm - remove hard_coded defaults which are different from proxmox or will be changed later by proxmox (https://github.com/ansible-collections/community.general/pull/850).
```

```suggestion
- proxmox - remove hard-coded defaults which are different from proxmox or could be changed later by proxmox (https://github.com/ansible-collections/community.general/pull/850).
- proxmox_kvm - remove hard-coded defaults which are different from proxmox or could be changed later by proxmox (https://github.com/ansible-collections/community.general/pull/850).
```
I think you should also write a bit more text what this concretely means for users. Also, I think it would be good to list all defaults that have been removed ("The defaults removed were ``disk=3``, ``cores=1``, ...") so that users have a good way of seeing what could actually affect their use-cases.

---

Fixed in https://github.com/ansible-collections/community.general/pull/850/commits/200985b76b8e5518de710a970e854ab28261a0a0 and https://github.com/ansible-collections/community.general/pull/850/commits/454ffdba20b63dda56065ff38357d29bc5bc1d6e.